content/blog/developer-experience.md

```diff
@@ -0,0 +1,81 @@
+leveraging these built-in capabilities. 
+
+## Software Development with Git
+Let's assume your company follows the `feature branches git workflow`:
+* Developers work on individual `feature` branches.
+* The `develop` branch serves as the integration point where all features are merged to validate the application in the staging environment.
+* Once the software receives the green light, the code is released to the `main` branch and deployed to the production environment.
+
+![feature branches git workflow](/blog/images/git-workflow.png)
+
+Don't be surprised, but the Orchestrator plugin automatically installs all the needed resources to handle these steps for you.
+
+## A Reference Architecture for Automated Deployments of Serverless Workflows
+The proposed reference architecture consists of the following components:
+* `Orchestrator Helm chart`: the installer of RHDH Orchestrator .
+* `RHDH`: the Red Hat product for Backstage.
+* `Tekton/Red Hat OpenShift Pipelines`: the recommended CI pipeline to build images and deployment configurations.
+* `ArgoCD/Red Hat OpenShift GitOps`: the selected CD pipeline to deploy the workflow on the RHDH instance.
+* `Quay`: the container registry service to store the software images.
+* `SonataFlow platform`: the SonataFlow implementation of the Serverless Workflow specifications, including a Kubernetes operator and the platform services (data index, jobs service).
+* `SonataFlow`: the custom resource representing the workflow.
+* `GitHub workflow repo`: the source code repository of the workflow.
+* `GitHub config repo`: the repository of the `kustomize` deployment configuration.
+
+![feature branches git workflow](/blog/images/cicd-architecture.png)
+ 
+Please note that all these components are either bundled with the Orchestrator plugin or managed by the software projects generated with the RHSH Software Templates.
+
+## The Software Development Lifecycle
+### Creating the Software Project
+RHDH offers the [software template](https://backstage.io/docs/features/software-templates/) functionality to create the foundational structure of
+software projects adhering to industry best practices in software development and deployment. 
+
+The Orchestrator plugin comes with its own templates designed to kickstart your workflow project. By selecting a template tagged with `orchestrator`, you gain access to the following benefits, all at no cost:
+
+* A fully operational software project to develop your serverless workflow, in a newly generated Git repository under the organization of your choice.
+* A ready-to-use configuration repository with a [kustomize](https://kustomize.io/) configuration to deploy the workflow on the designated RHDH instance.
+* (*) Automated CI tool deployment to trigger workflow builds on the selected cluster.
+* (*) Automated CD automation deployment to deploy applications implementing your workflow.
+
+(*): optional but highly recommended!
```

Comment:
Maybe you can few words to explain why they are recommended

Reply:
Again, instead of spending too many words with that (e.g., the real answer is "because it has the OpenShift/K8s operator to install it), I reviewed this statement a bit. Please take a look